draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2116,7 +2116,281 @@ uCDN-A                         dCDN-B                         dCDN-C
+		ObjectList is a metadata object describing lists of objects that can be used in the context of CI/T v2 trigger spec, trigger status resources and other contexts as required. The ObjectList object can either embed the lists of objects or point to external URL(s) that hold such lists.  ObjectList allows specification of an object list type, providing instructions on the interpretation of the object list format.</t>  
+
+	<t>
+		ObjectLists MAY be recursive, i.e. including references to secondary manifests, including references to HLS, MPEG-DASH or HSS manifests as well as additional JSON-encoded ObjectLists , etc. The party consuming the object list MUST parse all recursions based on the object list type property. When doing so, the consuming party should also detect potential loops when descendant  ObjectList points back to parent  ObjectList.
+	</t>
+	<t>
+ObjectLists MAY combine regular objects and secondary ObjectLists in the same object.
+		Please note that when embedding non-JSON object lists directly in ObjectList, absolute URLs MUST be provided at all times and the text SHOULD be encoded in accordance with the JSON grammar specification [ECMA404], including explicit newline encoding. When the uCDN accesses ObjectList metadata resources published by the uCDN, the same interface authentication and authorization requirements would apply, as when accessing the interface itself.
+	</t>
+
+	<t>ObjectList is encoded as an array of per-object records in JSON format as follows</t>
+	<t> External HLS manifest:</t>
+<figure><artwork><![CDATA[ 
+[
+  { 
+    “href”:"https://example.com/hls/a36f764e/index.m3u8",
+    "type":"hls" 
+  }
+]
+]]></artwork></figure>
+
+	<t>External object list in text format:</t>
+<figure><artwork><![CDATA[ 
+[
+  { 
+    “href”:“https://example.com/hls/35cdc008/assets”,
+    "type":“text" 
+  }
+]
+]]></artwork></figure>
+
+
+	<t>List of external manifests and objects: of mixed types</t>
+<figure><artwork><![CDATA[ 
+[
+  { 
+    “href”: “https://example.com/hls/35cdc008/index.m3u8”, 
+    “type":“hls" 
+  },
+  { 
+    “href”:“https://example.com/dash/35cdc008/main.mpd”, 
+    “type":“dash"
+  },
+  { 
+    “href”:“https://example.com/dash/35cdc008/files.json”, 
+    “type":“json" 
+  }
+]
+]]></artwork></figure>
+
+	<t>Embedded JSON-encoded object list</t>
+<figure><artwork><![CDATA[ 
+[
+  { 
+    “data”: [
+      { 
+	“href”: “https://example.com/hls/35cdc008/index.m3u8”, 
+	“type": “hls" 
+      },
+      { 
+	“href”:“https://example.com/dash/35cdc008/main.mpd”,
+	“type": “dash"
+      },
+      { 
+	“href”:“https://example.com/img/35cdc008/thumb-l.jpg”, 
+	“size”: 10260
+      },
+      { 
+	“href”:“https://example.com/img/35cdc008/thumb-s.jpg”,
+	“size”: 1453
+      }
+    ],
+    “type”: “json”
+  }
+]
+]]></artwork></figure>
+
+	<t>Embedded HLS manifest</t>
+<figure><artwork><![CDATA[ 
+[
+  {  
+    "data”: “#EXTM3U\n
+    #EXT-X-STREAM-INF:BANDWIDTH=150000,RESOLUTION=416x234\n
```

Comment:
Since this is a literal string, should the HLS playlist be indented with the JSON?

Reply:
I'm not sure if I stated this clearly enough.  My concern here was really around the fact that this is a multi-line string, and the fact that on the second and subsequent lines, the indention could be interpreted as being embedded into the string on each line.  Without researching it, I'm not sure how HLS manifests would react to seeing their lines indented with whitespace, which is why I was wondering if lines after the first one should have no indention (or maybe even formatted without line breaks, although that's very unreadable).

Reply:
Understood now
I have removed all indentation